# Array of detections for a single frame
Header header
Detection[] detections
uint32 total_detections
float32 inference_time_ms
string model_name
